DTS-STN/canadian-dental-care-plan
# AB#3335 -- Added contact us to apply for your child screen

### Description
Added contact us to apply for your child screen (if under 16 with a child)

### Related Azure Boards Work Items
[AB#3335](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3335)

### Screenshots (if applicable)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/93342d63-107d-4f19-b9d6-148e729c5280)

### Checklist

### Test Instructions
Select applying for myself/child and enter dob before 16 years of age to confirm new page

### Additional Notes
n/a
